tests/amazon/aws/triggers/test_batch.py

```diff
@@ -176,3 +180,90 @@ async def test_batch_trigger_exception(mock_response):
     task = [i async for i in trigger.run()]
     assert len(task) == 1
     assert TriggerEvent({"status": "error", "message": "Test exception"}) in task
+
+
+def test_batch_sensor_trigger_serialization():
+    """
+    Asserts that the BatchSensorTrigger correctly serializes its arguments
+    and classpath.
+    """
+    trigger = BatchSensorTrigger(
+        job_id=JOB_ID,
+        region_name="eu-west-1",
+        aws_conn_id="airflow_test",
```

Comment:
Same; these 2 could be module level constants.

Reply:
Done

Reply:
changed in sensors missed the trigger added now